public/index.html

```diff
@@ -37,23 +37,23 @@
   
 <body>
   
-    <nav>
-      <div class="text_container">
-        <div class="nav_primary">
-          <img src="assets/dscl_logo.png">
-        </div>
+  <nav>
+    <div class="text_container">
+      <div class="nav_primary">
+        <img src="assets/dscl_logo.png">
```

Comment:
add `alt="logo"` to make it accessible

Reply:
Alt text uses:
1. Adding alternative text to photos is first and foremost a principle of web accessibility. Visually impaired users using screen readers will be read an alt attribute to better understand an on-page image.

2. Alt tags will be displayed in place of an image if an image file cannot be loaded.

3. Alt tags provide better image context/descriptions to search engine crawlers, helping them to index an image properly.

Checkout https://moz.com/learn/seo/alt-text#:~:text=Adding%20alternative%20text%20to%20photos,image%20file%20cannot%20be%20loaded.